.vscode/settings.json

```diff
@@ -12,5 +12,7 @@
     "nodelib",
     "OSMC",
     "redeclaration"
-  ]
+  ],
+  "files.insertFinalNewline": true,
+  "files.trimTrailingWhitespace": true,
```

Comment:
I think @AnHeuermann meant that you can add this to your VS code settings so the code you write is always formatted without trailing whitespaces. I don't think he meant to add these settings to the project.

Reply:
Indeed 😆

Reply:
Surely it would be a good idea to add these to the project settings anyways, right? Doing that would help to ensure a more consistent style across contributors.

Reply:
Not everyone uses VS Code, so I usually don't add the .vscode directory. For a VS Code extension this might be different.
Both is fine with me.